test-ar.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
```

Comment:
should this be file here? maybe add it to the test scripts folder in a separate PR if you think it should be pushed

Reply:
accidentally included this file - removed!